File: frontend/src/app/model/socketMessagePart.ts
export class SocketMessagePart {

  id: string;
  senderName: string = "";
  receiverName: string = "";
  messagePart: string = "";

  partNumber: any;
  totalParts: number;
  port: number;

  constructor(id: string, messagePart: string, senderUsername: string, receiverUsername: string, partNumber: any, totalParts: number) {
    this.id = id;
    this.messagePart = messagePart;
    this.senderName = senderUsername;
    this.receiverName = receiverUsername;
    this.partNumber = partNumber;
    this.totalParts = totalParts;
  }
}

File: backend/src/main/java/net/etfbl/backend/model/SocketMessagePart.java
package net.etfbl.backend.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class SocketMessagePart {

  private String id;
  private String senderName;
  private String receiverName;
  private String messagePart;

  private Long partNumber;
  private Long totalParts;
  private Long port;

}
